src/components/Footer/index.tsx

```diff
@@ -1,16 +1,13 @@
-import * as S from './style';
-import useDarkState from 'Stores/useDarkStore';
+'use client';
 
-const Footer = ({ isDark }: { isDark: boolean }) => {
-  // const { isDark, setIsDark } = useDarkState();
+import * as S from './style';
 
-  return (
-    <div>
-      <S.FooterContainer id="footer">
-        <S.FootText>Copyright 2023. the moment All rights reserved.</S.FootText>
-      </S.FooterContainer>
-    </div>
-  );
-};
+const Footer = () => (
+  <div>
+    <S.FooterContainer id="footer">
+      <S.FootText>Copyright 2023. the moment All rights reserved.</S.FootText>
+    </S.FooterContainer>
+  </div>
```

Comment:
```suggestion
  <S.FooterContainer id="footer">
    <S.FootText>Copyright 2023. the moment All rights reserved.</S.FootText>
  </S.FooterContainer>
```

불 필요한 태그가 존재하는 듯 합니다.

Reply:
확인 감사드립니다! commit suggestion 하겠습니다!